pact4s/src/test/scala/org/broadinstitute/dede/workbench/leonardo/provider/LeoProvider.scala

```diff
@@ -0,0 +1,169 @@
+package org.broadinstitute.dede.workbench.leonardo.provider
```

Comment:
this seems to have the wrong path, `dede` should be `dsde`

Reply:
Sure thing. That should be updated now. Thanks for catching that.

Reply:
Thanks Ivan! That seemed to do the trick.

Reply:
Thanks! I believe that the appropriate changes have been ported over. Please let me know if I seem to be missing anything.